automates/text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndExport.scala

```diff
@@ -32,8 +33,9 @@ object ExtractAndExport extends App {
 
   val config = ConfigFactory.load()
 
-  val inputDir: String = ""
-  val outputDir: String = ""
+  val numOfWikiGroundings: Int = config[Int]("apps.numOfWikiGroundings")
+  val inputDir: String = "/Users/alexeeva/Desktop/automates-related/SuperMaaS-sept2021/cosmos-jsons-beautified"
```

Comment:
This has been long overdue. Been reluctant to disrupt my existing workflow (input/output dirs defined in the script instead of loaded from config) :D Fixing it now.